src/core/component.js

```diff
@@ -203,6 +203,7 @@ Component.prototype = {
     if (!this.initialized) {
       this.init();
       this.initialized = true;
+      if (el.isPlaying) { this.play(); }
```

Comment:
The confusing part is that `isPlaying` implies that it's already playing, and therefore we wouldn't need to call `play()`.